package.json

```diff
@@ -27,7 +27,7 @@
     "lint:clear-cache": "rimraf .eslintcache",
     "lint:dependencies": "depcheck && yarn dedupe --check",
     "lint:dependencies:fix": "depcheck && yarn dedupe",
-    "lint:eslint": "eslint . --cache",
+    "lint:eslint": "yarn ts-node ./scripts/run-eslint.ts --cache",
```

Comment:
Another update from the Core's Fix Eslint PR https://github.com/MetaMask/core/pull/5132